lib/compat/wordpress-6.1/class-wp-theme-json-data-gutenberg.php

```diff
@@ -56,4 +77,35 @@ public function get_data() {
 		return $this->theme_json->get_raw_data();
 	}
 
+	/**
+	 * Checks whether the registered blocks were already processed for this origin.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param string $origin Data source for which to cache the blocks.
+	 *                       Valid values are 'core', 'blocks', 'theme', and 'user'.
+	 * @return bool True on success, false otherwise.
+	 */
+	protected static function has_same_registered_blocks( $origin ) {
```

Comment:
Same with this one. 6.1.0 did not add this method to `WP_Theme_JSON_Data`. I'll revert this change too.